docs/assessment/index.md

```diff
@@ -22,35 +22,60 @@ Each assessment is a single directory in the `assessments` folder or any subfold
+| `uuid`                                                            | string  | [Unique identifier](../uuid.md). (Required; no default)                                                                                   |
+| [`type`](#assessment-types)                                       | string  | Either `"Homework"` or `"Exam"`. (Required; no default)                                                                                   |
+| `title`                                                           | string  | The title of the assessment (e.g., `"Derivatives and integrals"`). (Required; no default)                                                 |
+| `set`                                                             | string  | Which assessment set this belongs to (e.g., `"Quiz"`, `"Practice Quiz"`). (Required; no default)                                          |
+| [`number`](#assessment-naming)                                    | string  | The number of the assessment within the set (e.g., `"1"`, `"2B"`). (Required; no default)                                                 |
+| [`module`](#grouping-assessments-by-modules)                      | string  | The module that this assessment belongs to (e.g., `Chapter 3`). (Optional; default: `"Default"`)                                          |
+| [`allowAccess`](../accessControl/index.md)                        | array   | List of access rules. (Optional; default: no student access)                                                                              |
+| [`zones`](#question-specification)                                | array   | Specification of zones and questions. (Optional; default: none)                                                                           |
+| `text`                                                            | string  | HTML text shown on the assessment overview page. (Optional; default: none)                                                                |
+| `multipleInstance`                                                | boolean | Whether to allow students to create whole new attempts at the entire assessment. (Optional; default: `false`)                             |
+| `maxPoints`                                                       | number  | The maximum points that can be earned. (Optional; default: sum of zone max points)                                                        |
+| [`maxBonusPoints`](#assessment-points)                            | number  | The maximum number of additional points that can be earned beyond `maxPoints`. (Optional; default: 0)                                     |
+| [`shuffleQuestions`](#question-order-randomization)               | boolean | Whether to randomize the question order. (Optional; default: `false`)                                                                     |
```

Comment:
The default actually depends on the assessment type. It's "False" for homework, "True" for exam.